crates/gateway-client/src/lib.rs

```diff
@@ -61,10 +47,6 @@ pub trait GatewayApi: Sync {
         unimplemented!();
     }
 
-    async fn state_update_deprecated(&self, block: BlockId) -> Result<StateUpdate, SequencerError> {
-        unimplemented!();
-    }
-
     async fn state_update_with_block(
```

Comment:
Now that the names are free..

```suggestion
    async fn block(
```

Reply:
Well, I'm not quite sure. `block` is not a great description of what this method does?

Reply:
It gets the block's data :D? Ofcourse a bit confusing given the history, but if we were just starting fresh now I would have just called this method get `block` 🤷 No big deal though.

Maybe `full_block` then?

Reply:
It doesn't even get _all_ data for the block. For example, signature's not included...